.ci/validation.sh

```diff
@@ -116,10 +116,19 @@ no-error-pmd)
   echo "CS_version: ${CS_POM_VERSION}"
   mkdir -p .ci-temp/
   cd .ci-temp/
+  git clone https://github.com/checkstyle/build-tools.git
+  cd build-tools
+  git checkout checkstyle_7417
+  PMD_POM_VERSION=$(mvn -e -q -Dexec.executable='echo' -Dexec.args='${project.version}' \
```

Comment:
build-tools is where the configuration lies. We have to install that and used it in PMD's run to make breaking changes.